test/e2e_node/critical_pod_test.go

```diff
@@ -44,11 +43,14 @@ var _ = SIGDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:CriticalPod]
 	f := framework.NewDefaultFramework("critical-pod-test")
 	ginkgo.Context("when we need to admit a critical pod", func() {
 		ginkgo.It("[Flaky] should be able to create and delete a critical pod", func() {
-			configEnabled, err := isKubeletConfigEnabled(f)
-			framework.ExpectNoError(err)
-			if !configEnabled {
-				e2eskipper.Skipf("unable to run test without dynamic kubelet config enabled.")
-			}
+
+			//TODO (SergeyKanzhelev): https://github.com/kubernetes/kubernetes/issues/106939
```

Comment:
It used to be feature-gated, and dynamic config was used to enable the feature for the test. Looks like the removal PRs missed this part:
- Removed the feature gate: https://github.com/kubernetes/kubernetes/pull/80342/files#diff-94cfb00d75a10103bd0a3476b3db35446dbde4c4050913210445023abad3ae38L51
- Removed the call to tempSetCurrentKubeletConfig: https://github.com/kubernetes/kubernetes/pull/105482/files#diff-94cfb00d75a10103bd0a3476b3db35446dbde4c4050913210445023abad3ae38L47

Reply:
yes, thank you